content/en/docs/tasks/tls/manual-rotation-of-ca-certificates.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Manual rotation of CA certificates
+content_template: templates/task
+reviewers:
+- 
+---
+
+{{% capture overview %}}
+
+This page shows how to manually rotate certificate authority (CA) certificates.
+
+{{% /capture %}}
+
+
+{{% capture prerequisites %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+For more information about authentication in Kubernetes, see "[Authenticating](/docs/reference/access-authn-authz/authentication)".
+For more information about best practices for CA certificates, see "[Single root CA](docs/setup/best-practices/certificates/#single-root-ca)".
```

Comment:
These lines could move to a new What's next section at the foot of the page. Do you think they work better there or here?
If they stay, I'd write:
```suggestion
For more information about authentication in Kubernetes, see [Authenticating](/docs/reference/access-authn-authz/authentication/).
For more information about recommended practice for CA certificates, see [Configure certificates manually](https://kubernetes.io/docs/setup/best-practices/certificates/#configure-certificates-manually).
```

Reply:
I guess they are good here, for they give some precontext of the story.

Reply:
@sftim added almost all your suggestions with the latest commit.

Reply:
I'd write:
```suggestion
For more information about authentication in Kubernetes, see [Authenticating](/docs/reference/access-authn-authz/authentication/).
For more information about recommended practice for CA certificates, see [Configure certificates manually](https://kubernetes.io/docs/setup/best-practices/certificates/#configure-certificates-manually).
```